build/Targets/Packages.props

```diff
@@ -151,7 +151,7 @@
     <SystemConsoleVersion>4.3.0</SystemConsoleVersion>
     <SystemDiagnosticsContractsVersion>4.3.0</SystemDiagnosticsContractsVersion>
     <SystemDiagnosticsDebugVersion>4.3.0</SystemDiagnosticsDebugVersion>
-    <SystemDiagnosticsFileVersionInfoVersion>4.3.0</SystemDiagnosticsFileVersionInfoVersion>
+    <SystemDiagnosticsFileVersionInfoVersion>4.0.0</SystemDiagnosticsFileVersionInfoVersion>
```

Comment:
For us, I think the risk is very low. As far as I can tell, we don't take a dependency on anything in the higher versions. However, this could be a bigger deal for VS if our version is considered the "canonical" version for all of VS.

Now that we're on the new project system I may be able to back this change out.